### DISCUSSION TITLE
combat d'équipe

### DISCUSSION THREAD

**Discussion by kekeishere** [Crownicles/Crownicles#1789] [Suggestions fermées]
Genre ce que l'on pourrait faire c'est que par exemple chaque semaine pendant un seul jour a une certaine heure (ex: 00h00 et fin 23h59 il y'a un "ennemi qui vas pop dans la map et tout le monde vas recevoir cette information en faisait un prochain mini event avec le lieu et le temps restant et tout le monde doit se rendre au lieu (ex: foret celestrum) et doit attaquer le boss disons qu'il a 100 000 de pv et que chaque joueur peut lui lancer 1 seule attaque une fois l attaque faite le joueur s'enfuit et trace sa route biensur le joueur peut revenir affronter le boss et ainsi de suite et ce combat la il sera sous la forme d'un rapport

exemple
Vous arrvez enfin a la foret celestrum et devant vous se dresse une enorme gorille du kilimanjaro de plus de 10metre vous lui tournez autour et vous avez la possibilité de lui lancer une attaque dans un angle mort
⚔️ attaque de rat 🔥 attaque avec une torche 🐖 jet de cochon 👊 coup de poing

Vous avez réussit a lui invliger une attaque de rat -253pv

**Comment by BastLast** [comment]
Une feature relativement proche de celle ci est sur la roadmap donc je passe cela en refusé

**Reply by kekeishere** [reply]
oké
